Src/static/scripts.js

```
@@ -93,13 +95,99 @@ function updateFeedPreference(toggle) {
+    set enabled(value) {
+        this._enabled = Boolean(value);
+        this.saveLimiterState(this._enabled);
+    }
+
+    get limitValue() {
+        return this._limitValue;
+    }
+
+    set limitValue(value) {
+        const limit = parseInt(value, 10);
+        if (isNaN(limit) || limit < 1) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)